Add explanation how to use metadata repository locally (#306)
oracle/graalvm-reachability-metadata: README.md

If you are a library or framework maintainer, the best way to make your code ready for GraalVM Native Image is to provide reachability metadata as part of your JARs.
Please visit [this web page](https://www.graalvm.org/latest/reference-manual/native-image/metadata/) for more information.

> ## How to test/use this repo locally

Reviewer: ```suggestion
## How to Test or Use This Repository Locally
```

> In order to test Reachability Metadata Repository locally on your project or with your additional changes, you should do the following:
> * Clone metadata repository
> ```shell
> git clone git@github.com:oracle/graalvm-reachability-metadata.git
> ```
> * Use Native Build Tools with the plugin you prefer:
>   * [Gradle](https://graalvm.github.io/native-build-tools/latest/gradle-plugin.html#metadata-support)
>   * [Maven](https://graalvm.github.io/native-build-tools/latest/maven-plugin.html#metadata-support)

Reviewer: ```suggestion
You can test the reachability metadata from this repository locally against your project or with additional changes.
First, you clone the repository:
```shell
git clone git@github.com:oracle/graalvm-reachability-metadata.git
```
And then second, you adjust the configuration for the metadata support in either the [Maven plugin](https://graalvm.github.io/native-build-tools/latest/maven-plugin.html#metadata-support) or [Gradle plugin](https://graalvm.github.io/native-build-tools/latest/gradle-plugin.html#metadata-support).
```
